### DISCUSSION TITLE
Style modifications

### DISCUSSION THREAD

**Discussion by abdullahO2** [organicmaps/organicmaps#2575] [Ideas]
Is it possible to allow modification to the style and the way the map is displayed, and to be according to the user’s desire, so that the user can show elements of a higher zoom level, or change the way an element appears, and I also suggest creating a gallery for the modified styles, and providing a way to share, apply and navigate between them while browsing the map .

Thank you and best regards

**Comment by biodranik** [comment]
We plan to support several different styles. What is your use-case? Can you provide any examples in other apps?

**Comment by abdullahO2** [comment]
osmand styles:
https://docs.osmand.net/docs/user/map-legend/
https://github.com/osmandapp/OsmAnd-resources/tree/master/rendering_styles

mapsforge RenderTheme:
https://github.com/mapsforge/mapsforge/blob/master/docs/Rendertheme.md
https://forum.locusmap.eu/index.php?board=40.0
https://www.openandromaps.org/en/legend/elevate-mountain-hike-theme##

**Comment by darylbond** [comment]
I would also appreciate this feature, in particular increasing the visibility of roads in remote areas at lower zoom levels. At the moment the default style means that roads are invisible at a zoom level that makes sense for the large distances being viewed (think remote regions of Australia).
